naplib/naplab/process_ieeg.py

```diff
@@ -50,7 +50,7 @@ def process_ieeg(
     """
```

Comment:
Change the default for "aud_channel_infer_method" to be something else depending on which you think is more robust: https://github.com/naplab/naplib-python/pull/125/files#diff-8230fda0643e3dce44dca7b4de9ef4b5caf8fd69f73ad8f5572681f01a948c56R36